SECURITY.md

```diff
@@ -2,20 +2,24 @@
 
 ## Supported Versions
 
-Use this section to tell people about which versions of your project are
-currently being supported with security updates.
-
-| Version  | Supported                   |
-| -------- | --------------------------- |
-| 0.24.x   | :white_check_mark:          |
-| 0.23.x   | Depending on usage/severity |
+The Hickory DNS team fully supports the most recent major release and will consider patches to prior
```

Comment:
I think "major release" is potentially confusing here because traditionally "major" is used to describe the `x` component in `x.y.z` when in the case of Hickory (for the foreseeable future) we're talking about `y`.